android/java/res/layout/fragment_secure_password.xml

```diff
@@ -25,20 +24,17 @@
             android:layout_marginBottom="18dp"
             android:text="@string/create_password"
             android:textColor="@color/wallet_text_color"
-            android:lineSpacingExtra="2sp"
-            android:textSize="22sp"
-            />
+            android:textStyle="bold" />
 
         <TextView
+            style="@style/DefaultRegular"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:gravity="center_horizontal"
             android:layout_marginHorizontal="16dp"
             android:layout_marginBottom="40dp"
             android:text="@string/create_password_subtitle"
-            android:textColor="@color/wallet_onboarding_secondary_text_color"
-            android:lineSpacingExtra="4sp"
-            android:textSize="14sp"/>
+            android:textColor="@color/wallet_onboarding_secondary_text_color" />
```

Comment:
can we replace the color with nala colors ? same with all other colors we have in the PR.

Reply:
Sorry @deeppandya, I missed this as the color code indicated by the Figma design does not match the Nala color code :/

The variation is negligible, so I'm proceeding with the substitution. 
Ideally though, the color code for `text_secondary` should match in Nala and in Figma otherwise it creates **a lot** of confusion.

<img width="1521" alt="image__19__png_and_image__20__png_and_Downloads" src="https://github.com/brave/brave-core/assets/3308503/099df3d2-6ce0-42b6-8bd8-d9d9d428d67d">

Reply:
@aguscruiz we need to have color names/values aligns.